aspnetcore/fundamentals/host/generic-host.md

```diff
@@ -307,9 +307,9 @@ webBuilder.UseSetting(WebHostDefaults.HostingStartupExcludeAssembliesKey, "assem
 
 The HTTPS redirect port. Used in [enforcing HTTPS](xref:security/enforcing-ssl).
 
-**Key**: https_port
-**Type**: *string*
-**Default**: A default value isn't set.
+**Key**: https_port<br>
```

Comment:
@tdykstra Curious ... is this a change in guidance? Two spaces provides a rendered return. I've been using the two-space approach from the start (introduced to me back in my .NET Core docs days), but I can stop that and start using bare returns if that's preferred.

Reply:
I can see how a tag might be better ... it's easier to :eyes: where they are given transparent spaces.

Reply:
I didn't know why some lines were breaking at end of line and others weren't. If I were starting a doc from scratch I would avoid invisible formatting cues like double spaces, but since that's what was done in this doc already I'll change the tags to double spaces, maybe add a comment explaining it.

Reply:
Here's the CommonMark spec for it. I guess we could have been a bit more obvious with the single `\` at the end of a line.

https://spec.commonmark.org/0.28/#hard-line-breaks

Yes, you'll see this in use across the docs. IIRC, both Scott and I were using it. It's in common use on the .NET Core docs, too, where it was originally brought to my attention.

Reply:
Thanks for reviewing this pr and pointing that out!